packages/toolpad-app/src/runtime/ToolpadApp.tsx

```diff
@@ -91,13 +91,13 @@ import api, { queryClient } from './api';
 
 const browserJsRuntime = getBrowserRuntime();
 
-const isPreview = process.env.NODE_ENV !== 'production';
-const isCustomServer = process.env.TOOLPAD_CUSTOM_SERVER === 'true';
-const isRenderedInCanvas =
+const IS_RENDERED_IN_CANVAS =
```

Comment:
We've typically been using all caps for values exposed from `constants`
```suggestion
const isRenderedInCanvas =
```

Reply:
We typically use all caps for static values that are know at compile time. It tells the call site that this variable abstracts a constant value, and could be inlined without changing semantics if we wanted to.